### ISSUE TITLE
AnimationDepthBB11

### ISSUE THREAD

**Issue by rsryman** [rryman/BeneshForBalletBook4#40]
Re animations on pages 30, 32, 34, 36, 40 & 43:

As discussed by phone Aug 2/17, the algorithm used in DanceForms animations distorts the "depth" so that it is difficult to accurately see the location of the dancer on the column from downstage to upstage. The closer to upstage the larger the dancer, so we have decided to shift the figure upstage. The farthest upstage location we will use is the location midway between upstage and a middle stage, as in the starting position on p. 30. The BMN and animation on this page will remain as is. 

Robyn will change the BMN stage location signs and corresponding stage plan arrows on the pages where we currently use farthest upstage location (pp. 32 & 43). Rhonda will shift animations so that the upstage-most location the dancer reaches is the location midway between upstage and a middle stage, as in the starting position on p. 30.

**Comment by rsryman** [comment]
**NOTE FOR RHONDA:** In DanceForms, the upstage-most location is 4 and the downstage-most location is -4. I had initially used 2 as the location "midway between middle and upstage." I have checked the shifted p. 30 animation and note that Robyn considers the dancer's placement on 3 to be visually acceptable as "midway between middle and upstage." I will therefore shift all locations so that the upstage-most location is 3. 

**NOTE FOR ROBYN:** We agreed that you will watch the shifted animations and see if the downstage-most locations are visually acceptable with the current notation.

If there is anything I have missed or misunderstood, please call asap. I'll start revising the animations now.

**Comment by rsryman** [comment]
Uploaded files with dimensions & zoom as on p. 30 BB11TravelFwdRectanglep30.m4v:

Yesterday, Aug 2/17: 
BB11TravelBackwardWalkp32.m4v (in the book you uploaded Aug 3)
BB11TravelSideRightChPdBp34.m4v (NOT YET in the book you uploaded Aug 3)
BB11TravelSideLeftp36.m4v (NOT YET in the book you uploaded Aug 3)

Today, Aug 3/17: 
BB11TravelDiagonalp41.m4v (en face)
BB11TravelDiagonalp43.m4v (facing corner)

**To be discussed.** The p. 30 stage plan shows a rectangular shape and the animation creates that shape by doing 3 chassé-coupé actions in depth and 4 in width. The p. 42 stage plan shows a rectangular shape (according to what you visually perceive?) but the animation actually travels as much upstage-downstage as it does stage right-left, as on p 40. Same on p. 32.

I thought this mismatch was because the dimensions of the DF stage don't match the BMN plans, but not so because I created matching stage proportions when I started animating these geometric shapes. Could it be because the stage is rectangular (wider than deep) but there are the same number of BMN divisions/increments (8) in width as in depth, so each width increment is greater than each depth increment? None of the readers had a problem with the square pattern re Backward walks, yes?
